www/templates/diary/detail.html

```diff
@@ -33,7 +33,7 @@
         <div style="height: 15px;"></div>
         <div class="row" style="padding-top: 10px !important; padding-bottom: 10px !important; padding-left: 20px !important; padding-right: 20px !important;">
           <div class="col" style="width: 30%; margin-top: 0; font-size: 1.1em; font-weight: thin;">
-          <p style="font-size: 1.1em; color: #cccccc; margin-bottom: 0;">Distance</p>{{getFormattedDistance(trip.properties.distance)}} km</div>
+          <p style="font-size: 1.1em; color: #cccccc; margin-bottom: 0;">Distance</p>{{getFormattedDistance(trip.properties.distance)}} mi</div>
```

Comment:
Yes, the graph was in m/s which is now mph https://github.com/e-mission/e-mission-phone/pull/260/files#diff-30e353982546b2071becab3038f1bc08R89